arm-asazure/2016-05-06-preview/swagger/asazure.json

```diff
@@ -0,0 +1,489 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-05-06-preview",
+    "title": "ASAzureWebAPI",
+    "description": "The Azure Analysis Services API provides a RESTful set of web services enables users to create, retrieve, update, and delete servers"
```

Comment:
The Azure Analysis Services API provides a RESTful set of web services enables **that** users to create, retrieve, update, and delete servers